src/ansible_navigator/actions/_actions.py

```diff
@@ -36,12 +36,19 @@
 
 
 def _import(package: str, action: str) -> None:
-    """Import the given action file from a package"""
+    """Import the given action from a package.
+
+    :param package: The name of the package
+    :param action: The action to import
+    """
     importlib.import_module(f"{package}.{action}")
 
 
 def _import_all(package: str) -> None:
-    """Import all actions in a package"""
+    """Import all actions in a package.
+
+    :param package: The name of the package.
```

Comment:
Did we reach an agreement on sentences with trailing periods vs phrases in params? Looks like even within this PR it's inconsistent.

Reply:
I've been trying to leave it off, thx